### ISSUE TITLE
`@Ignore` inspection tag is ignored

### ISSUE THREAD

**Issue by daFreeMan** [rubberduck-vba/Rubberduck#2668]
RD 2.0.11 Access 2010

I have 2 variables that are declared to store off application state. They are passed to a module that sets them, then later they're used to restore application state.

The first issue, which I believe is noted & being worked on at some point, is that the code inspections flag these as never being assigned. Since this is a future fix/enhancement, I let RD add the `'@Ignore UnassignedVariableUsage` flag to the code so the inspections wouldn't pick it up again. Unfortunately, the `'@Ignore` directive is being ignored...

![rd ignore ignored](https://cloud.githubusercontent.com/assets/11889733/22786267/cfa1f3dc-eea5-11e6-9efc-cdacb0a35104.PNG)

As you can see, `ProtectStructure` and `ProtectWindows` are declared and immediately passed `ByRef` to `ToggleWorkbookProtection` which sets them. Later in this proc, `ToggleWorkbookProtection` is called again, with `ProtectTheBook:=True` to return the workbook protections settings to where they were.

Despite the `'@Ignore UnassignedVariableUsage` tags on the lines above their declaration, the code inspection is still flagging them.

**Comment by daFreeMan** [comment]
This also raises the possibility of a new inspection: `PassingAllParametersInOrderByName`. That's really just overkill, isn't it... :/

**Comment by comintern** [comment]
I can't replicate this in the current build...
```
Public Sub Caller()
'@Ignore UnassignedVariableUsage
    Dim var As Long
    Callee bar:=var
End Sub

Public Sub Callee(ByRef bar As Long)
    bar = 42
End Sub
```
...works fine. I think this ended up getting fixed along with #2525 and\or #2523.
